scripts/validate-token-configs.ts

```diff
@@ -0,0 +1,391 @@
+import { ethers } from "ethers";
+import axios from "axios";
+
+/*
+ * =============================
+ * Section: Types and Constants
+ * =============================
+ */
+interface ChainInfo {
+  symbol: string;
+  name: string;
+  axelarChainId: string;
+  tokenAddress: string;
+  tokenManager: string;
+  tokenManagerType: TokenManagerType;
+}
+interface TokenInfo {
+  tokenId: string;
+  deployer: string;
+  originalMinter: string | null;
+  prettySymbol: string;
+  decimals: number;
+  originAxelarChainId: string;
+  tokenType: string;
+  deploySalt: string;
+  iconUrls: {
+    svg: string;
+  };
+  deploymentMessageId: string;
+  coinGeckoId: string;
+  chains: ChainInfo[];
+}
+type TokenManagerType = (typeof tokenManagerTypes)[number];
+
+// Constants
+const tokenManagerTypes = [
+  "nativeInterchainToken",
+  "mintBurnFrom",
+  "lockUnlock",
+  "lockUnlockFee",
+  "mintBurn",
+  "gateway",
+];
+const ITSAddress = "0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C";
+const COINGECKO_API_KEY = "CG-3VGxh1K3Qk7jAvpt4DJA3LvB";
+const COINGECKO_URL = "https://api.coingecko.com/api/v3/coins/";
+const CHAIN_CONFIGS_URL =
```

Comment:
testnet validations will fail on this. can these be environmentalized? try using the `@axelarjs/api/getAxelarConfigs` import instead of hard-coding the download like this